spec/support/capybara.rb

```diff
@@ -1,4 +1,5 @@
 require "selenium/webdriver"
+require "chromedriver-helper"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.